modular_skyrat/modules/gun_quirk/quirk.dm

```diff
@@ -0,0 +1,52 @@
+///Mail goodie list.
+#define GUN_MAIL_LIST list(/obj/item/gun_maintenance_supplies, \
+	/obj/item/bedsheet/patriot, \
+	/obj/item/clothing/mask/gas/hecu2, \
+	/obj/item/storage/backpack/industrial/cin_surplus/random_color, \
+	/obj/item/toy/gun, \
+	/obj/item/toy/ammo/gun, \
+)
+
+///Gun you get on spawn.
+#define STARTER_GUN_LIST list(/obj/item/gun/ballistic/automatic/pistol/g17, \
+	/obj/item/gun/ballistic/automatic/pistol/mk58, \
+	/obj/item/gun/ballistic/automatic/pistol/pdh/peacekeeper, \
+)
+
+/datum/quirk/item_quirk/gun_nut
+	name = "Gun Nut"
+	desc = "You own a firearm, presumably for self defense or boasting about how cool it is; and you thought it's such a good idea to bring it on station with you."
+	icon = FA_ICON_PERSON_RIFLE
+	value = 8
+	quirk_flags = QUIRK_HUMAN_ONLY | QUIRK_HIDE_FROM_SCAN
+	gain_text = span_notice("Man, I sure do love guns!")
+	lose_text = span_danger("Maybe having a gun isn't that a good idea...") //man so stupid
+	medical_record_text = "Patient has passed a psychological evaluation and has been authorised with firearms and ammunition handling." //shouldn't show up anyways so whatever
```

Comment:
Nobody has authorised this person to have a firearm, a weapons permit is not implied nor should be granted solely based on this quirk.